File: codesignal/arcade/Intro/JS/isLucky.js
function isLucky(n) {
    let s = n.toString()
    
    let i = 0;
    let j = s.length - 1
    
    let sum = 0
    let sum2 = 0
    
    while(i < j){
        sum += parseInt(s[i])
        sum2 += parseInt(s[j])
        i++
        j--
    }
    
    return sum == sum2
}

File: codesignal/arcade/Intro/Csharp/isLucky.cs
bool isLucky(int n) {
    string nStr = Convert.ToString(n);
    
    int i = 0;
    int j = nStr.Length - 1;
    
    int sum = 0;
    int sum2 = 0;
    
    while(i < j){
        sum += (int)nStr[i];
        sum2 += (int)nStr[j];
        i++;
        j--;
    }
    return sum == sum2;
}
